modules/combined/tests/phase_field_fracture_viscoplastic/tests

```diff
@@ -0,0 +1,9 @@
+[Tests]
+  [./crack2d]
+    type = 'Exodiff'
+    input = 'crack2d.i'
+    exodiff = 'crack2d_out.e'
+    abs_zero = 1e-09
+    use_old_floor = True
+  [../]
+[../]
```

Comment:
I suggest adding a newline here (Atom does that automatically and it will avoid future issues like the one Kyle reported on the ML a few minutes ago)